extensions/pyRevitTools.extension/pyRevit.tab/Modify.panel/edit2.stack/Flip.pushbutton/script.py

```diff
@@ -1,25 +1,49 @@
 """Provides options for flipping (Hand/Face) selected elements."""
 
-from pyrevit import revit, DB
+from pyrevit import revit, DB, script
 from pyrevit import forms
 
 
 __context__ = 'selection'
 
+def can_flip_wall(el):
```

Comment:
I've added these small functions to check which options are relevant with current selection (see below)